runtime/src/accounts.rs

```diff
@@ -658,6 +659,14 @@ pub fn create_test_accounts(
     }
 }
 
+pub fn update_accounts(accounts: &Accounts, pubkeys: &[Pubkey], slot: u64) {
+    for pubkey in pubkeys {
+        let amount = thread_rng().gen_range(0, 10);
+        let account = Account::new(amount, 0, &Account::default().owner);
```

Comment:
how about randomizing `account.data` for the hashing benchmark case? Is it enought currently for us to only focus on normal accounts with no data?

Reply:
yea , it would be interesting to have that.

Reply:
I'll add this in a follow-up PR.